src/SFA.DAS.EmployerIncentives.Web/Services/Applications/ApplicationService.cs

```diff
@@ -40,10 +40,20 @@ public async Task<ApplicationConfirmationViewModel> Get(string accountId, Guid a
             response.EnsureSuccessStatusCode();
 
             var data = await JsonSerializer.DeserializeAsync<GetApplicationResponse>(await response.Content.ReadAsStreamAsync());
-            
+
             return MapFromGetApplicationResponse(data);
         }
 
+        public async Task Confirm(string accountId, Guid applicationId)
+        {
+            var request = MapToPostRequest(applicationId, accountId);
+
+            using var response = await _client.PostAsJsonAsync($"/accounts/{request.AccountId}/confirm-application/{applicationId}", request);
```

Comment:
I think this endpoint is going to change to a PUT at the /accounts/{accountId}/applications/{applicationId} endpoint.

Reply:
i was told to use PATCH